webapp/views/wpt-results.js

```diff
@@ -99,6 +100,9 @@ class WPTResults extends WPTColors(WPTFlags(PathInfo(LoadingState(TestRunsUIBase
         opacity: 0.7;
         box-shadow: 5px 5px 5px;
       }
+      td[selected] {
+        border: 2px solid #000000;
```

Comment:
Note to myself: I have seen missing parts of borders before but couldn't reproduce today. Check again